src/olympia/reviewers/views.py

```diff
@@ -1114,21 +1114,6 @@ def allow_resubmission(self, request, **kwargs):
             status_code = status.HTTP_409_CONFLICT
         return Response(status=status_code)
 
-    @drf_action(
-        detail=True,
-        methods=['post'],
-        permission_classes=[GroupPermission(amo.permissions.REVIEWS_ADMIN)],
-    )
-    def clear_pending_rejections(self, request, **kwargs):
```

Comment:
Drive-by cleanup to reduce confusion - This API was already obsolete since 9f1e462ad03708cb7516b64aa08fcaf538028b0a